tests/common/helpers/dut_ports.py

```diff
@@ -0,0 +1,17 @@
+def encode_dut_port_name(dutname, portname):
+    return dutname + '|' + portname
+
+
+def decode_dut_port_name(dut_portname):
+    tokens = dut_portname.split('|')
+    if len(tokens) >= 2:
+        dutname = tokens[0]
+        portname = tokens[1]
+    elif len(tokens) == 1:
+        dutname = None
+        portname = dut_portname
+    else:
+        dutname = None
+        portname = None
```

Comment:
Will we ever have this case? because you always set to unknown|unknown in most of the failure cases

Reply:
I think so. Place here just incase the metadata file got human edited or corrupted.